packages/integrations/preact/src/server.ts

```diff
@@ -27,19 +27,17 @@ async function check(
 	useConsoleFilter();
 
 	try {
-		try {
```

Comment:
I only refactored this part as the nested `try` isn't necessary. Before it's `try { try..catch } finally {}`. It can be flatten as `try..catch..finally` and work the same.